src/components/hand-poses.js

```diff
@@ -0,0 +1,124 @@
+const POSES = {
+  open: "allOpen",
+  thumbDown: "thumbDown",
+  indexDown: "indexDown",
+  mrpDown: "mrpDown",
+  thumbUp: "thumbsUp",
+  point: "point",
+  fist: "allGrip",
+  pinch: "pinch"
+};
+
+AFRAME.registerComponent("hand-poses", {
+  schema: {
+    leftPose: { type: "string", default: "allOpen" },
+    rightPose: { type: "string", default: "allOpen" },
+    mixer: { type: "string" },
+    gltfEntity: { type: "string", default: "a-gltf-entity" }
+  },
+
+  init() {
+    this.animatePose = this.animatePose.bind(this);
+    this.animatePoses = this.animatePoses.bind(this);
+    this.firstUpdate = this.firstUpdate.bind(this);
+    if (this.el.querySelector(this.data.mixer)) {
+      this.mixer = this.el.querySelector(this.data.mixer).components["animation-mixer"];
+    }
+
+    let onLoad;
+    onLoad = () => {
+      if (this.el.querySelector(this.data.mixer)) {
+        this.mixer = this.el.querySelector(this.data.mixer).components["animation-mixer"];
+      }
+      if (!this.mixer) {
+        // Can't find the mixer until spawned into the scene.
+        // TODO: Figure out the right event to listen to to get this done.
+        window.setTimeout(onLoad, 100);
+        return;
+      }
```

Comment:
Maybe the events are more sane after #125

Reply:
If you just put this on the element with the mixer all of this (and the selector) shuould go away. The mixer is on RootScene so a new mixer, and new hand-poses will be created each time an avatar is loaded.